server/network.js

```diff
@@ -420,3 +458,53 @@ network.listNetwork_ = function(networkPath) {
   });
   return ret;
 };
+
+/**
+ * Find all edges connecting the gene to other exist genes.
+ * @param {string} file Network file path.
+ * @param {!Array<string>} genes Genes to add to the graph.
+ * @param {!Array<!network.Node>} nodes Nodes that already in the network.
+ * @return {{
+ *   edges: !Array<!network.Edge>
+ * }} Edges between the new gene and network.
+ * @private
+ */
+network.incrementalEdges_ = function(file, genes, nodes) {
+  var oldNodes = {};
+  nodes.forEach(function(node) {
+    oldNodes[node.id] = true;
+  });
+  var newNodes = {};
+  genes.forEach(function(gene) {
+    if (!(gene in oldNodes)) {
+      newNodes[gene] = true;
+    }
+  });
+  var edges = [];
+  var result = network.readNetwork_(file);
+  result.edges.forEach(function(edge) {
+    if ((edge.source in newNodes && edge.target in oldNodes) ||
+      (edge.source in oldNodes && edge.target in newNodes) ||
+      (edge.source in newNodes && edge.target in newNodes)) {
+      edges.push(edge);
+    }
+  });
+  return {
+    edges: edges
+  };
+};
+
+/**
+ * Gets the node info for the network.
+ * @param {string} file File path of the network.
+ * @return {{
+ *   nodes: !Array<!network.Node>
+ * }} The node info.
+ * @private
+ */
+network.networkInfo_ = function(file) {
```

Comment:
Would it be better to name this as "networkNodes" if you are simply retrieving the nodes?